content/_index.en.md

```diff
@@ -24,4 +24,8 @@ usecases:
     title: Explore the Rules
     description: Model any kind of government rule, explore them and evaluate who is eligible to what.
     caption: (Exploring the Rules) is a web interface to explore the rules in New Zealand's Openfisca built from government legislation, regulation and some policies.
+
+  legexp_nsw:
+    caption_1: uses OpenFisca and its
```

Comment:
This sentence doesn't complete

Reply:
Thanks @Br3nda ! Take a look at the html code and the image posted in the description: it is counter-intuitive, but those variables are used to build a sentence in the final html.